build.sbt

```diff
@@ -778,7 +760,7 @@ lazy val scalaDist = Project("scala-dist", file(".") / "target" / "scala-dist-di
     ),
     publishArtifact in (Compile, packageSrc) := false
   )
-  .dependsOn(libraryAll, compiler, scalap)
+  .dependsOn(library, reflect, compiler, scalap)
```

Comment:
Does this remove xml, swing and parser combinators from the default repl (in the distribution archive)? I guess it's fine, but would be finer if we had a good way to add things to the default repl's classpath..